### PR TITLE
Enable side comments on EA forum (as opt in)

### PR DESCRIPTION
**Pull request by Will-Howard**
Co-authored-by: "Jonathan Mustin" <jcmustin@gmail.com>

The only problem we could find with side comments was that they introduce a horizontal scroll on mobile, so we just hide the button on mobile. This UI is required for the highlights project which will use the same toolbar

This PR makes side comments opt in, we could also just enable them fully if everyone. I don't anticipate any further changes before enabling them so this probably makes sense, but I thought people might want to try it on staging/prod before committing to enabling it fully

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1203962583336773) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/components/comments/CommentOnSelection.tsx [hunk 1]
```diff
@@ -18,6 +19,11 @@ const selectedTextToolbarStyles = (theme: ThemeType): JssStyles => ({
     "&:hover": {
       background: theme.palette.panelBackground.darken08,
     },
+
+    // Hide on mobile to avoid horizontal scrolling
+    [theme.breakpoints.down('xs')]: {
+      display: isEAForum ? "none" : "initial",
```
Comment (jpaddison3) [inline packages/lesswrong/components/comments/CommentOnSelection.tsx:25]: Wait, does LW seriously not want this? Also, how does this relate to: https://github.com/ForumMagnum/ForumMagnum/pull/6514

File: packages/lesswrong/lib/betas.ts [hunk 2]
```diff
@@ -22,7 +22,7 @@ const isEAForum = forumTypeSetting.get() === 'EAForum'
 // Features in progress                                                     //
 //////////////////////////////////////////////////////////////////////////////
 
-export const userHasCommentOnSelection = isEAForum ? disabled : shippedFeature;
+export const userHasCommentOnSelection = isEAForum ? optInOnly : shippedFeature;
```
Comment (jpaddison3) [inline packages/lesswrong/lib/betas.ts:25]: I woulda sworn this was optInOnly on the Forum already. TBH, at this point I would ship it. The feature is ~complete on LW, and seems net positive. There's not a lot of value in putting things in optInOnly unless you know what you're looking for, IMO.

**Review by jpaddison3 — APPROVED** [review]
I have a question, that I'm kinda lazily not looking into, but do want someone to answer for me before merging.

File: packages/lesswrong/lib/betas.ts (hunk 2 above)
Comment (Will-Howard) [reply packages/lesswrong/lib/betas.ts]: Viewing side comments was opt-in, but making them was disabled... which overall meant they were just disabled

Comment (jpaddison3) [reply packages/lesswrong/lib/betas.ts]: Not quite! Because of block quotes. But now I understand.

**Comment by jpaddison3** [comment]
Think about Jim's comment, but you have a :shipit: from me

File: packages/lesswrong/components/comments/CommentOnSelection.tsx (hunk 1 above)
Comment (jpaddison3) [reply packages/lesswrong/components/comments/CommentOnSelection.tsx:25]: I indeed was not tracking that distinction and it is helpful

**Pull request merged**